src/main/java/net/nicoulaj/idea/markdown/editor/MarkdownPreviewEditor.java

```diff
@@ -138,6 +145,31 @@ public void handleSettingsChanged(@NotNull final MarkdownGlobalSettings newSetti
         jEditorPane.addHyperlinkListener(new MarkdownLinkListener(jEditorPane, project, document));
     }
 
+    private void resetStyleSheet(HTMLEditorKit kit) {
+        final StyleSheet style = new StyleSheet();
+        String customStylesheetPath = MarkdownGlobalSettings.getInstance().getCustomStylesheetPath();
+        URL styleSheetResource = null;
+        if (customStylesheetPath != null && customStylesheetPath.trim().length() > 0) {
```

Comment:
Are utility methods like [Strings.isNullOrEmpty](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/Strings.html#isNullOrEmpty(java.lang.String)) not available?